### PR TITLE
Add explicit reference to coding style guidelines in docs #207

### LINKED ISSUE #207: Add explicit reference to coding style guidelines in docs
See title, very important with new devs coming in. Right now I can't find it easily from the `README.md` starting point.
Changing `DeveloperGuide.md` is probably a good choice.

### PR DESCRIPTION
**Pull request by kychua**
Fixes #207

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@kychua submitted v1 for review.

* [v1 1/1] [DeveloperGuide: Add links to coding standards](https://github.com/se-edu/addressbook-level4/pull/297/commits/a98809031ef5949ecfb9321c436ac4ba0dc3f522)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/d741336a16662b675c117ed2aa3ade11a934f24c...a98809031ef5949ecfb9321c436ac4ba0dc3f522))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/297/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -72,6 +72,15 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 * Reason: Required libraries may not have been downloaded during the project import.
 * Solution: [Run tests using Gradle](UsingGradle.md) once (to refresh the libraries).
 
+### 1.5. Coding style
+
+We use the following coding standards for this project:
+
+* [Java coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)
+* [CSS coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Css.html)
+* [GitHub Flavored Markdown coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Gfmd.html)
```
Comment (PierceAndy) [inline docs/DeveloperGuide.md]: I don't think it's necessary to repeat the words `coding standard` for each line.

**Comment by CanIHasReview** [comment]
# v2

@kychua submitted v2 for review.

* [v2 1/1] [DeveloperGuide: Add links to coding standards](https://github.com/se-edu/addressbook-level4/pull/297/commits/1ec4efd6de7cc3d96dcd3731e24fa908ce39743a)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/d741336a16662b675c117ed2aa3ade11a934f24c...1ec4efd6de7cc3d96dcd3731e24fa908ce39743a)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level4/compare/b41345b7ceb81de275b62807c72ef61931f06377~1...b41345b7ceb81de275b62807c72ef61931f06377))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/297/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -72,6 +72,15 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
...
 
+### 1.5. Coding style
+
+We use the following coding standards for this project:
+
+* Java: https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html
+* CSS: https://oss-generic.github.io/process/codingStandards/CodingStandard-Css.html
+* GitHub Flavored Markdown: https://oss-generic.github.io/process/codingStandards/CodingStandard-Gfmd.html
+
+Following a consistent coding style makes the code more readable and maintainable, so please ensure your code follows the above coding standards.
```
Comment (damithc) [inline docs/DeveloperGuide.md]: This sentence can be omitted. Devs are supposed to know this.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -72,6 +72,15 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 * Reason: Required libraries may not have been downloaded during the project import.
 * Solution: [Run tests using Gradle](UsingGradle.md) once (to refresh the libraries).
 
+### 1.5. Coding style
+
+We use the following coding standards for this project:
+
+* Java: https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html
+* CSS: https://oss-generic.github.io/process/codingStandards/CodingStandard-Css.html
+* GitHub Flavored Markdown: https://oss-generic.github.io/process/codingStandards/CodingStandard-Gfmd.html
```
Comment (damithc) [inline docs/DeveloperGuide.md]: Shorten like this?

> Coding standards to follow: [Java](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html), [CSS](), [GFMD]()

Or make it future proof, like this:

> We follow [oss-generic coding standards](https://github.com/oss-generic/process/blob/master/docs/CodingStandards.md).

Comment (PierceAndy) [inline docs/DeveloperGuide.md]: I think making it future proof would be the wiser decision.

**Comment by CanIHasReview** [comment]
# v3

@kychua submitted v3 for review.

* [v3 1/1] [DeveloperGuide: Add link to coding standards](https://github.com/se-edu/addressbook-level4/pull/297/commits/0f027b45600f074c676ec89bd1ab9e60d31e18f8)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/d741336a16662b675c117ed2aa3ade11a934f24c...0f027b45600f074c676ec89bd1ab9e60d31e18f8)) ([:chart_with_upwards_trend: Interdiff between v2 and v3](https://github.com/se-edu/addressbook-level4/compare/62626b1d2cc2570864c2d00b38ba659c29f6465b~1...62626b1d2cc2570864c2d00b38ba659c29f6465b))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/297/3/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Pull request merged**